
sniped from ChromiumOS

https://bugs.gentoo.org/424707

From 154f7861d19a2bd5c79c488f9989610971db451b Mon Sep 17 00:00:00 2001
From: Han Shen <shenhan@google.com>
Date: Thu, 31 May 2012 16:49:35 -0700
Subject: [PATCH] Fixed dbus-c++ gcc 4.7 building problem.

Just add "unistd.h" inclusion to eventloop-integration.h.

BUG=None
TEST=Manually built using gcc 4.7.

Change-Id: I87bd1f90db6a4c974a5ed8134044e8be2034aff2
Reviewed-on: https://gerrit.chromium.org/gerrit/24260
Tested-by: Han Shen <shenhan@google.com>
Reviewed-by: Yunlian Jiang <yunlian@chromium.org>
Commit-Ready: Han Shen <shenhan@chromium.org>
Upstream-Status: Inappropriate [ not author, no upstream ]
Signed-off-by: Thilo Cestonaro <thilo.cestonaro@ts.fujitsu.com>
---
 include/dbus-c++/eventloop-integration.h |    1 +
 1 file changed, 1 insertion(+)

Index: libdbus-c++-0.9.0/include/dbus-c++/eventloop-integration.h
===================================================================
--- libdbus-c++-0.9.0.orig/include/dbus-c++/eventloop-integration.h
+++ libdbus-c++-0.9.0/include/dbus-c++/eventloop-integration.h
@@ -26,6 +26,7 @@
 #define __DBUSXX_EVENTLOOP_INTEGRATION_H
 
 #include <errno.h>
+#include <unistd.h>
 #include "api.h"
 #include "dispatcher.h"
 #include "util.h"
